translations/de/task_description.html

```diff
@@ -1,33 +1,30 @@
 <p>
-    In mathematics, particularly in linear algebra,
-    a skew-symmetric matrix(also known as an antisymmetric or antimetric) is a square matrix <strong>A</strong> whose transpose
-    is also its negative.
-    This means it satisfies the equation
-    <span style="white-space: nowrap">A = &minus;A<sup>T</sup>.</span>
-    If the entry in the i-th row and j-th&nbsp;column is a<sub>ij</sub>, i.e.
-    <span style="white-space: nowrap">A = (a<sub>ij</sub>)</span>
-    then the symmetric condition becomes
-    <span style="white-space: nowrap">a<sub>ij</sub> = &minus;a<sub>ji</sub>.</span>
+  In der Mathematik, insbesondere in der linearen Algebra, ist eine schiefsymmetrische
```

Comment:
Is it possible to drop the last "in"?
_In der Mathematik, insbesondere der linearen Algebra, ist..._